README.rst

```diff
@@ -52,7 +45,23 @@ This is optional. You can set your own return url by adding this to `settings.py
 
     PESAPAL_OAUTH_CALLBACK_URL = 'app_name:url_name'  # this needs to be a reversible
 
-#. Run `python manage.py syncdb` to create the polls models.
+The url can have the 'merchant_reference' passed to it as a parameter for further processing.
+Or be without a parameter.
+
+for example the url pattern
+
+r'^response/$'
+
+without the merchant reference parameter
+
+or
+
+r'^response/(?P<merchant_reference>\d+)/$'
```

Comment:
Oh, I added it as an example that the dev is to implement it in their project urls conf. The url's name is later set in the config PESAPAL_TRANSACTION_DEFAULT_REDIRECT_URL to be reversed here, https://github.com/metatrevor/django-pesapal/blob/sandbox/django_pesapal/views.py#L32. Now it depends on the dev's preference to process it further or just redirect the end user to some view.